watertap/unit_models/nanofiltration_DSPMDE_0D.py

```diff
@@ -2015,6 +2021,139 @@ def calculate_scaling_factors(self):
             sf = iscale.get_scaling_factor(self.flux_mol_phase_comp[t, x, p, j])
             iscale.constraint_scaling_transform(con, sf)
 
+        for (t, x, p, j), con in self.eq_interfacial_partitioning_feed.items():
+            hfd = value(self.hindrance_factor_diffusive_comp[t, j])
```

Comment:
If I understand the logic here, why wouldn't this be `partition_factor_steric_comp` instead?

Reply:
Good question...I guess I need more coffee -fixed in next pr.